source/dbo/Stored Procedures/Reusable Notification/uspGenerateReusableNotification.sql

```diff
@@ -266,34 +265,33 @@ BEGIN TRY
 			,td.StayLengthInMonths1							AS 'MonthsTravelled1'
 			,dbo.ufnGetCountryName(td.Country2Id)			AS 'TravelCountry2'
 			,td.StayLengthInMonths2							AS 'MonthsTravelled2'
-			,dbo.ufnGetCountryName(td.Country3Id)			AS 'TravelCountry3' 
+			,dbo.ufnGetCountryName(td.Country3Id)			AS 'TravelCountry3'
 			,td.StayLengthInMonths3							AS 'MonthsTravelled3'
 			,vd.HasVisitor									AS 'ReceivedVisitors'
 			,vd.TotalNumberOfCountries						AS 'FromHowManyCountries'
 			,dbo.ufnGetCountryName(vd.Country1Id)			AS 'VisitorCountry1'
 			,vd.StayLengthInMonths1							AS 'DaysVisitorsStayed1' --NB is this captured in days in ETS? It's captured in months in NTBS
 			,dbo.ufnGetCountryName(vd.Country2Id)			AS 'VisitorCountry2'
 			,vd.StayLengthInMonths2							AS 'DaysVisitorsStayed2'
-			,dbo.ufnGetCountryName(vd.Country3Id)			AS 'VisitorCountry3' 
+			,dbo.ufnGetCountryName(vd.Country3Id)			AS 'VisitorCountry3'
 			,vd.StayLengthInMonths3							AS 'DaysVisitorsStayed3'
 			--comorbidities
-			,cod.DiabetesStatus								AS 'Diabetes' 
+			,cod.DiabetesStatus								AS 'Diabetes'
 			,cod.HepatitisBStatus							AS 'HepatitisB'
-			,cod.HepatitisCStatus							AS 'HepatitisC' 
+			,cod.HepatitisCStatus							AS 'HepatitisC'
 			,cod.LiverDiseaseStatus							AS 'ChronicLiverDisease'
 			,cod.RenalDiseaseStatus							AS 'ChronicRenalDisease'
 			,id.[Status]									AS 'ImmunoSuppression'
 			,dbo.ufnYesNo(id.HasBioTherapy)					AS 'BiologicalTherapy'
-			,dbo.ufnYesNo(id.HasTransplantation)			AS 'Transplantation' 
+			,dbo.ufnYesNo(id.HasTransplantation)			AS 'Transplantation'
 			,dbo.ufnYesNo(id.HasOther)						AS 'OtherImmunoSuppression'
 			,rfs.[Status]                                   AS 'CurrentSmoker'
 			--treatment details
-			,dbo.ufnYesNo(cd.IsPostMortem)					AS 'PostMortemDiagnosis' 
-			,dbo.ufnYesNo(cd.DidNotStartTreatment)			AS 'DidNotStartTreatment' 
+			,dbo.ufnYesNo(cd.IsPostMortem)					AS 'PostMortemDiagnosis'
+			,dbo.ufnYesNo(cd.DidNotStartTreatment)			AS 'DidNotStartTreatment'
 			--next two fields set in separate function later on
-			,NULL						                    AS 'ShortCourse' 
-			,NULL							                AS 'MdrTreatment' 
-			,cd.MDRTreatmentStartDate						AS 'MdrTreatmentDate' 
+			,cd.TreatmentRegimen							AS 'TreatmentRegimen'
```

Comment:
This needs to be converted from the 'code' held in clinical details to a user-readable string, e.g. StandardTherapy to Standard therapy, MdrTreatment to 'RR/MDR/XDR treatment'